recipes/chex/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "chex" %}
+{% set version = "0.0.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/chex-{{ version }}.tar.gz
+  sha256: 6bd16c7aae962a7a7dd66504c78b470fe0fc27d82c9a7fe73c24f7e6562f1896
+
+build:
+  number: 0
+  skip: true   # [win or py<36]
```

Comment:
It looks like `chex` is os-independent according to its `setup.py`. Are you excluding windows because of known issue or a missing dependency? Because if we can build this for windows, we can move this to a `noarch: python` recipe (I think -- I need to check `dataclasses` to see if it is a no-op for python >=3.7).

Reply:
An indirect dependency, jaxlib (chex->jax->jaxlib) is not available on windows.
I would hav eliked to make it noarch, but chex requires dataclasses, which should only be dependend upon in python 3.6, as it's part of the standard lib starting with python 3.7

Reply:
Yup - it looks like `dataclasses` no ops for the python versions where it isn't needed so you can drop the selector on it if necessary:

https://github.com/conda-forge/dataclasses-feedstock/blob/master/recipe/meta.yaml

Reply:
And about windows? can i make it noarch or i have to keep it with the skip?

Reply:
You don't have to worry about the missing dependency. See the third note under:
https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

The package itself is `noarch` and we let conda handle the dependency resolution.

Reply:
Done. Thank you.